python/idsse_common/test/test_grid_proj.py

```diff
@@ -64,11 +62,6 @@ def approx_tuple(values: Tuple[float, float]) -> Tuple:
 def grid_proj() -> GridProj:
     return GridProj.from_proj_grid_spec(EXAMPLE_PROJ_SPEC, EXAMPLE_GRID_SPEC)
```

Comment:
EXAMPLE_PROJ_SPEC is defined with a '+r=' instead of '+a='. It's fine to leave if you plan to fix in a different task.

Reply:
Thanks for catching that. Great example of how we need this corrected in all code and all files, otherwise it will keep finding ways to sneak back in.